scripts/screens/all_screens.py

```diff
@@ -85,6 +86,7 @@
 
 choose_mate_screen = ChooseMateScreen('choose mate screen')
 choose_mentor_screen = ChooseMentorScreen('choose mentor screen')
+choose_adoptive_parent_screen = ChooseAdoptiveParentScreen('choose adaptive parent screen')
```

Comment:
there is a incorrectly using word (maybe a typo) in the name "adaptive" vs "adoptive"

Reply:
thanks for mentioning, fixed it